content/en/docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -64,29 +64,40 @@ spec:
           gpu-vendor.example/example-gpu: 1 # requesting 1 GPU
 ```
 
-## Clusters containing different types of GPUs
```

Comment:
I think we'd want to keep, but modify this heading.  Maybe "Manage clusters with different types of GPUs".  My logic is that this is in the task section and starting headings with a verb, when possible, is best.

I'd also keep the rather generic info from the original L#69-71 because it mentions a simple mechanism for managing the diff devices--labels.  Then, under this heading, identify the original content as "manual":

> ## Manage clusters with different types of GPUs
> 
> If different nodes in your cluster have different types of GPUs, then you
> can use [Node Labels and Node Selectors](/docs/tasks/configure-pod-container/assign-pods-nodes/)
> to schedule pods to appropriate nodes.
> 
> ### Manually labeling nodes
> 
> You can manually apply labels to nodes and then reference the labels in your pod specification to place workloads on the nodes with the specific type of GPU.
> 
> ```console
> # Label your nodes with the accelerator type they have.
> kubectl label nodes node1 accelerator=example-gpu-x100
> kubectl label nodes node2 accelerator=other-gpu-k915
> ```
> 
> That label key `accelerator` is just an example.
> You can use a different label key.

The proposed "Automatically labeling nodes..." heading would need another #, I think.